src/inverseRobot.js

```diff
@@ -14,17 +14,30 @@
  * Example:
  *
  * const kolli = { Kolli: 'name', 123: 'chipVer', 3: 'wheels' };
+ * const kolli = { Kolli: 'name', 123: 'chipVer', 3: 'wheels' };
+ *
+ * const robert = { Robert: 'name', 123: 'chipVer', 113: 'chipVer' };
  * const robert = { Robert: 'name', 123: 'chipVer', 113: 'chipVer' };
- * inverseRobot(robert) === null
- * inverseRobot(kolli) === { name: 'Kolli', chipVer: '123', wheels: '3' }
  *
+ * inverseRobot(kolli) === { name: 'Kolli', chipVer: '123', wheels: '3' }
+ * inverseRobot(robert) === null
  *
  * @param {object} robot
  *
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
```

Comment:
```suggestion
  const newRobot = {};
```
[NAMING] - use proper names for variables - if you want to name variable result, pick a better one :)